castervoice/rules/apps/editor/unity.py

```diff
@@ -0,0 +1,64 @@
+from dragonfly import Key,MappingRule, Dictation, Choice,Text
+from castervoice.lib.merge.state.short import R
+from castervoice.lib.merge.additions import IntegerRef
+from castervoice.lib.ctrl.mgr.rule_details import RuleDetails
+
+class UnityRule(MappingRule):
+	mapping = {
+		"<window>" : R(Key("c-%(window)s")),
+		"search <searchable> [<dictation>]" : R(Key("c-%(searchable)s,c-f") + Text("%(dictation)s")),
+		"clear <searchable> [search]" : R(Key("c-%(searchable)s,c-f,backspace")),
+		"console" : R(Key("cs-c")),
+
+		"max view" : R(Key("s-space")),
+
+		"(play | stop) game" : R(Key("c-p")),
+		"pause game" : R(Key("cs-p")),
+
+		"build and run" : R(Key("c-b")),
+		"build settings" : R(Key("cs-b")),
+
+		"make empty" : R(Key("as-n")),
+		"add component" : R(Key("cs-a")),
+
+		"align with view" : R(Key("cs-f")),
+		"move to view" : R(Key("ca-f")),
```

Comment:
Super settle, but how do you feel about renaming these to "view align" and "view move"? I'm kind of torn myself on the question.

Reply:
Personally I don't mind slightly more verbose commands for things that are not used too often if it makes them more memorable, descriptive and consistent. The commands are the same as the description you find in the unity menus. Saying just "view move" makes it sound like you're moving the view itself and would conflict with other commands for interacting with views whatever that would be.

If anything I think the commands I picked should be the official commands in this case, but we could add a shorter alias. Although that is also something individual users can do themselves.